doc/classes/CanvasItem.xml

```diff
@@ -340,13 +340,13 @@
 		<method name="get_global_transform" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns the global transform matrix of this item.
+				Returns the global transform matrix of this item with respect to its canvas layer.
 			</description>
 		</method>
 		<method name="get_global_transform_with_canvas" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns the global transform matrix of this item in relation to the canvas.
+				Returns the global transform matrix of this item with respect to the viewport. This is the same as get_canvas_transform() * get_global_transform(()
```

Comment:
> viewport

This is inconsistent with how you changed canvas to canvas layer above. Is this intended?

Reply:
@fHachenberg Bump :slightly_smiling_face: